ale/drivers/base.py

```diff
@@ -195,13 +264,51 @@ def detector_center(self):
     def spacecraft_name(self):
         return self._label['MISSION_NAME']
 
+    @property
+    def starting_detector_line(self):
+        return 1
+
+    @property
+    def starting_detector_sample(self):
+        return 1
+
+    @property
+    def detector_sample_summing(self):
+        return 1
```

Comment:
Should we be parsing something here and then failing to 1 (with a warning?)

Reply:
That was the behavior before refactoring, I imagine any problem with the implementation of the methods themselves can be done in a different PR. We still need to figure out the when where and how of warnings and errors.

Reply:
Summing is largely a mission by missions value. There is a lot of variation in what key it is stored in and what different values mean.

I think defaulting to 1 in the base class is a good choice.